### DISCUSSION TITLE
no respose after enter the commands from Using the Identity Endpoint

### DISCUSSION THREAD

**Discussion by kocoman2** [GerryDazoo/Slinger#479] [General]
no respose after enter the commands from
Using the Identity Endpoint

curl --location --request POST 'https://secure.sling.com/slingaccounts/rest/v3/identitypoint' --header 'Content-Type: application x-www-form-urlencoded' --data-urlencode 'context=SLINGIT_PORTAL' --data-urlencode 'policy_code=MAS' --data-urlencode 'email_address=my-sbaccount@me.com' --data-urlencode 'password=my-password'

**Comment by cliffsadler** [comment]
I am assuming that sbaccount@me.com is really your slingbox account login email, and the password is your slingbox account password.
when I tried it, I got Error: read ECONNRESET  with the no response message.  Perhaps this is shut down now.......

**Reply by kocoman2** [reply]
so how to get it working? desolder and read bga spi chips? thx

**Comment by cliffsadler** [comment]
OK, success.  Endpoint is still up.
Just had to learn how to use Postman.  First time user.  Off to apply to Slinger

**Reply by kocoman1** [reply]
so can i connect new devices?

**Comment by larahs** [comment]
can anyone confirm using postman to obtain slingbox password is still working?  or did i miss something in postman?

just tried and received error on a parameter after setting value as "MAS" -    "error": "policy_code:This field is required."

**Reply by kocoman1** [reply]
(not sure if its down now), but before It would work but your password have to be **correct**, else the server does not respond with a bad password return error, it just timeouts.. which was most confusing in the beginning.  but I apply the password to my old av and tuner and it does not work, I guess a quiet update bricked it.. and I can't check what version is installed or jtag it, it does not work with jtagulator maybe it need some pull down/up resistors ??  I am no tech guy so I might throw it out.  I have a 500 IIRC in storage I have to find it and come back talk here later someday

**Reply by GerryDazoo** [reply]
What kind of Slingbox is it? Please post the server output when you try to run it. Also please post the command you're using to get your password and it's output. The syntax needs to be exact. If it's a bricked Solo then trying to use the JTAG port won't get you very far. We do not know how to revive it. 
I don't suppose you've blocked out going access to the sling servers where you're running the command?

**Reply by kocoman1** [reply]
its AV and Tuner, (tried jtagulator on the 20 pin or 16 or something and the 6 pins but didn't get any results, do I need some pull/up/down resistors ?

The password I got and tried but it says is wrong, another user in the issues told me is because is blocked (but the password correct) based on the returned bits


(and a PRO in storage somewhere IIRC, If I find it someday I will post back)

**Reply by GerryDazoo** [reply]
Make sure you have TCP port 5678 outgoing blocked for your slingbox and then do a factory reset. I suspect your box has been bricked. After the reset the password will be "admin"
Stop messing with JTAG there is nothing useful you will be able to do even if you do get it running.

**Reply by kocoman1** [reply]
so the brick is a soft brick?? hmm thats interesting let me try it again thx
